content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,14 @@
+## Cascade Learning {.page_break_before}
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and the predictions there-of to make an educated decision about the identified predictive patterns. 
+This assembly of multiple machine learning methods is called ensemble learning. In an ensemble learning workflow, one may use various parameters to maximize the learned information. 
+Sometimes a consensus approach is ideal to select predictive features from a group of independently running models, other times one may coordinate various learning methods in such a way that one model uses information from another model in augmenting its predictive capability. 
+Random forest algorithms are a great example of ensemble learning where a consensus of the features selected through many different and independent decision trees are considered and ranked to make a final ranked list of important features that are predictive of a certain outcome (e.g. disease) [@doi:10.1186/1472-6947-13-134, @doi:10.1023/A:1010933404324].
+However, successful application of consensus based ensemble learning algorithms prefer what is known as "gold standard" data. These generally refer to well labeled cases where the diagnosis or label of a data point in the training dataset has very little uncertainty (or "noise") associated with it [@doi:10.1093/jamia/ocw028]. 
+In most cases of rare disease, due to the inherent nature of being less defined, the symptoms as well as any underlying biology comes with a reasonable amount of uncertainty (or "noise") leading to a "silver standard" dataset[@doi:10.1109/TNNLS.2013.2292894, @doi:10.1093/jamia/ocw028].
+This kind of data begets modifications in ensemble learning methods where one method leverages findings from another method to capture stable patterns existing in the "silver standard" data and reduce uncertainty.
+Such methods fall into the category of cascade learning.
+An example of cascade learning that showed robustness in view of noisy data was implemented to identify rare disease patients from electronic health records from the general population [@pmid:30815073].
```

Comment:
This is an interesting project, though that table (Table 1, where they pick some top-lists of patients to be evaluated) has self-assessment trap written all over it.

Reply:
:) yeah their patient assessment is not the best... but the case study still describes the principles of cascade learning which is why I decided to include this study here.. still looking to see if we can find other examples of work to cite but not coming across many cascade learning papers in rare disease :)